source/backups.txt

```diff
@@ -12,42 +12,222 @@ Backups
+outages. Once restored, you can simply point your application to the new 
+{+cluster+} to regain full functionality with complete read and write
+capabilities. We recommend developing automated deployment templates
+that strike a balance between availability and cost. However, your
+critical workloads might require multiple copies of snapshots in various
+locations.
+
+Recommendations for Backup Compliance Policy 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We recommend enforcing |service|'s Backup Compliance Policy to prevent
+unauthorized modifications or deletions of backups, thereby maintaining
+data integrity and supporting robust disaster recovery.  
+
+Recommendations for PIT Recovery
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|service|'s fully managed, customizable Cloud Backups and Continuous
+Cloud Backups enable precise Point In Time (PIT) recovery, minimizing
+data loss during failures. |service| can quickly recover to the exact
+timestamp before a failure event, giving you at least a one minute
+:abbr:`RPO (Recovery Point Objective)` and an :abbr:`RTO (Recovery Time
+Objective)` of less than 15 minutes when utilizing optimized restores,
+even in the event of the outage of the primary region. Recovery times
+can vary due to cloud provider disk warming and which point in time you
+are restoring to. If you can be flexible in your requirements for
+recovery, we recommend designing templates that identify the sweet spot
+between reasonable recovery options and cost. 
+
+Recommendations for Backup Costs 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To optimize |service| backup costs, you must adjust the backup frequency
+and retention policies to align with data criticality, reducing
+unnecessary storage expenses. You can also use incremental backups and
+built-in compression to minimize the amount of stored data. By selecting
+regions strategically for backup, you can avoid cross-region data
+transfer fees and choose the right cluster size based on workload to
+prevent overspending. By implementing these strategies, you can
+effectively manage costs while maintaining secure and reliable backups. 
 
 Examples
 --------
 
-The following examples <perform this action> using |service|
+The following examples backup and restore operations using |service|
 :ref:`tools for automation <arch-center-automation>`.
 
-These examples also apply other recommended configurations, including:
+These examples only apply for staging and production environments.
 
 .. tabs::
 
-   .. tab:: Dev and Test Environments
-      :tabid: devtest
+   .. tab:: CLI
+      :tabid: cli
 
-      .. include:: /includes/shared-settings-clusters-devtest.rst
+      Run the following command take a backup snapshot for the {+cluster+}
+      named myDemo and retain the snapshot for 7 days:
 
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
+      .. include:: /includes/examples/cli-example-backup-take-snapshot.rst
```

Comment:
I got this error when I ran this since backups weren't enabled first, can we add the command or prereq for that?: Error: https://cloud-dev.mongodb.com/api/atlas/v2/groups/60c8d499833523089e59f782/clusters/myCluster/backup/snapshots POST: HTTP 400 Bad Request (Error code: "BACKUP_JOB_NOT_FOUND") Detail: Backups are not enabled for this cluster. Reason: Bad Request. Params: []

Reply:
Will test and fix this.